F/FFTW/build_tarballs.jl

```diff
@@ -50,6 +50,9 @@ if [[ "${target}" == i686-w64-* ]]; then FLAGS+=( --with-incoming-stack-boundary
 # see https://sourceforge.net/p/mingw-w64/mailman/message/36287627/ for more info
 if [[ "${target}" == x86_64-w64-* ]]; then export CFLAGS="${CFLAGS} -fno-asynchronous-unwind-tables"; fi
 
+# On ppc64le, enable VSX
+if [[ "${target}" == powerpc64le-*  ]]; then FLAGS+=( --enable-vsx ); fi
```

Comment:
Should this be just `powerpc64*`?   Not sure if Julia support the bigendian POWER architectures?

Reply:
We don't support big endian as far as I am aware (or as far as anyone really wanted to try)